src/components/careers/layout.js

```diff
@@ -9,11 +9,16 @@ import Moment from "react-moment"
 import { FaFrown } from "react-icons/fa"
 import { formatDate } from "../../utils/utility"
 
-export default function CareersLayout({ children, noCarousel, cbetContent }) {
```

Comment:
So you added props... Can you add this to the PR description?? Like what new props you created and what the type they are supposed to be please. Also that would be helpful in the future PR requests.